modules/byo-vpc-prep.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * assemblies/byo-vpc.adoc
+
+[id="byo-vpc-prep_{context}"]
+= Preparing your BYO VPC
+
+.Procedure
+
+. Log in to your account on the link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html[AWS Management Console].
+. Create a VPC with the IP address of `10.0.0.0/16` and enable `dnsHostnames`.
```

Comment:
Is `10.0.0.0/16` required for the vpc? Can user set it to another value? If yes, we should't better mark it an example value. And there is no step to tell user how to **enabled `dnsHostnames`**

Reply:
@dofinn Can you reply to this one, as I am unsure of the answer.

Reply:
see `IPv4 CIDR block association restrictions` on https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html#add-cidr-block-restrictions

Reply:
`enableDNSHostnames` info can be found here: https://docs.aws.amazon.com/vpc/latest/userguide/vpc-dns.html#vpc-dns-support + https://docs.aws.amazon.com/glue/latest/dg/set-up-vpc-dns.html

Reply:
So, should I remove the IP addresses? I added a link to the AWS documentation around the DNS attributes for the VPC.

Reply:
@dofinn I added a link to the AWS documentation - https://deploy-preview-36198--osdocs.netlify.app/openshift-dedicated/latest/osd_cluster_create/byo-vpc.html#byo-vpc-prep_byo-vpc

Should I remove the explicit IP values or what exactly?

Reply:
I think the IP values are fine, they are standard AWS VPC spaces. . That section of the preview docs LGTM :), I would maybe hyperlink the DNS stuff. As a customer i know i wouldnt look at additional resources until i was at the end of the steps, but thats just my opinion.